### ISSUE TITLE
softlink in _checkouts folder?

### ISSUE THREAD

**Issue by tothlac** [inaka/cowboy-trails#106]
Hi,

As I see the examples folder contains a soft link to ../../. : https://github.com/inaka/cowboy-trails/blob/master/example/_checkouts/trails

Using a quite old version of rebar3 (3.13.3) sometimes `rebar3 as prod tar` works, sometimes don't (I don't know what causes the problem, on some machines it works, but on some other Jenkins agents we got this error):

```
 ===> error (eloop) accessing file /var/Jenkins/workspace/Upgrade_Release/our_app/_build/prod/lib/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/example/_checkouts/trails/.github
```
Looks like the problem is also related not only to rebar3 but to the OS / filesystem under the engine.
So basically the softlink pointing back to the root directory causes an infinite loop under some conditions.
Would it be possible to delete this softlink, or was it intentionally committed in : https://github.com/inaka/cowboy-trails/commit/ec2fe6ffc699a99bf2ae8cab8fb2ee9763cf7aad ?

**Comment by elbrujohalcon** [comment]
I think @paulo-ferraz-oliveira added it to be able to run tests with the latest version of the library.
If you can find a better way, go ahead and remove the softlink, @tothlac … If you create a PR, and the tests pass, I'll merge it and release a new version.

**Comment by elbrujohalcon** [comment]
OTOH… Your have a different issue: I don't know why it's doing that, but `rebar3`/`relx` should *not* include the `examples` folder in the release.

**Comment by paulo-ferraz-oliveira** [comment]
Yeah, the link is intentional as per https://github.com/inaka/cowboy-trails/pull/104/files#r779985965, but it should be Ok to merge a change if you find a better solution 😄
